README.Mac.md

```diff
@@ -1,294 +1,296 @@
+**b.** clone your fork to ~/Development/GitHub
 
+Paste the following lines in your terminal, and replace <your_username> with your github username before pressing enter.
 ```
-perl -v
+cd ~/Development/GitHub
+clone https://github.com/<your_username>/Loris.git
 ```
 
-## 1.4 MySQL 
-
-### 1.4.1 Check if MySQL is installed
-Check if you currently have MySQL installed:
-
+## 1.3 Install XCode Command Line Tools
+Type the following in your terminal and hit enter:
 ```
-mysql
+xcode-select --install
 ```
-If already installed continue to section 1.6 else continue
 
-### 1.4.2 Obtaining MySQL
-
-Download the latest DMG Archive for you current Mac setup (either 32-bit or 64-bit) from the following link [http://dev.mysql.com/downloads/mysql]
-(http://dev.mysql.com/downloads/mysql) and install the necessary files.
-
-### 1.4.3 Starting MySQL
-
-Open System Preferences on your machine and select the MySQL icon. Start the MySQL server if it hasn’t already been started. Check Automatically Start MySQL Server on Startup if you dont want to have to repeat this step everytime you resart your computer.
-
-### 1.4.4 Setting up root user
-
-Access MySQL with the following
+## 1.4 Install [Homebrew](https://brew.sh)
+Visit the official website of Homebrew and install by running their installer script in your terminal.
 
+## 1.5 Install Apache (a.k.a httpd)
+The following commands will install apache and launch it as a service on startup:
 ```
-mysql -u root
+brew update
+brew upgrade
+brew install httpd
+brew link httpd
+sudo brew services start httpd
 ```
 
-If the command is not found use the following 
-
+#### (useful) terminal commands for Apache
+How to start, stop, and restart apache.
 ```
-sudo sh -c 'echo /usr/local/mysql/bin > /etc/paths.d/mysql'
+sudo apachectl start
+sudo apachectl stop
+sudo apachectl -k restart
 ```
-Then restart terminal or open a new tab or window and start this step over.
 
-Once in the mysql command prompt, use the following command to reset the password
+### 1.5.1 Modify Apache Configuration
 
-``` 
-SET PASSWORD FOR 'root'@'localhost' = PASSWORD('$newpwd');
+Modify the apache configuration file for our development environment.
+```
+sudo open -e /usr/local/etc/httpd/httpd.conf
 ```
-<i>$newpwd ⇾ you new desired password for the root user</i>
-
-## 1.5 Smarty 3
-
-### 1.5.1 Obtaining Smarty 3
- 
-Go to [http://www.smarty.net/download](http://www.smarty.net/download) and download the latest stable release of Smarty 3.x.x
-
-### 1.5.2 Setting up Smarty 3
-
-Create the following directories if they do not already exist:
 
+**a.** Find the line:
 ```
-sudo mkdir /usr/local/lib/php
-sudo mkdir /usr/local/lib/php/Smarty
+Listen 8080
 ```
-
-Next, unzip the downloaded zip file and locate the libs directory.  Move it into the Smarty directory (created above), and rename it to smarty3 in the process. Use the following: 
-
+and replace to:
 ```
-sudo mv /path/to/Smarty-3.x.x/libs/ /usr/local/lib/php/Smarty/smarty3
+Listen 80
```

Comment:
This step isn't necessary and should be removed I think. Privileged ports (< 1024) can only be used by root and so this step requires apache to have root privileges which isn't a good idea.

Reply:
I think homebrew installs httpd as root. It's running as root on my development machine when I look at services running. I guess if I visited a malicious website some javascript could be used to poll my development server. I guess I have too much faith in Loris. I'll remove this line and add the port 8080. Thanks!